urdf2webots/parserURDF.py

```diff
@@ -421,52 +421,75 @@ def getSTLMesh(filename, node):
     return node
 
 
-def getOBJMesh(filename, node):
+def getOBJMesh(filename, node, link):
     """read obj file."""
     print('Parsing Mesh: ' + filename)
+    if hasattr(node, 'material') and node.material:
+        isVisual = True
+    else:
+        isVisual = False
+
     with open(filename, 'r') as file:
-        trimesh = node.geometry.trimesh
+        counter = 0
+        indexOffset = 0
         for line in file:
             header, body = line.split(' ', 1)
             if header == '#':
                 continue
+            elif header == 'o':
+                if counter != 0:
+                    if isVisual:
+                        link.visual.append(collision)
```

Comment:
collison is not defined here.

Reply:
that is the case for the first object, counter = 0, it will not append and create the collision object first. For the next 'o' it will then append it.

Reply:
Oh you're right, I was mislead by my linter (how also thought it was not defined).
Maybe this deserves a small comment, just to make it immediatly clear.

Reply:
I can initialize it before to make pep8 happy :p